query/control/controller_test.go

```diff
@@ -40,8 +41,12 @@ func init() {
 var (
 	mockCompiler = &mock.Compiler{
 		CompileFn: func(ctx context.Context) (flux.Program, error) {
+			// For Windows compatibility, sleep long enough for the system clock to update.
+			// See https://stackoverflow.com/a/57285684/16512381
+			time.Sleep(20 * time.Millisecond)
```

Comment:
I could make these sleeps conditional on `runtime.GOOS` if people would like

Reply:
I'm wondering if it's possible that sometimes 20ms isn't enough time and this would create flakyness. Would it be possible to do something along the lines of verifying that `time.Now()` is returning something different than when the waiting time start?

Reply:
I'd prefer as little conditionality as possible

Reply:
Looping until `time.Now()` changes seems to work (at least on Circle's executor), went with that.